### PR TITLE
Clarify project status in README

### PR DESCRIPTION
**Pull request by lucperkins**
Issue #32 indicates that we should make the current status of flake schemas more clear. This PR provides an admonition up top plus instructions on experimenting with schemas in the meantime.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,80 @@
...
-* [`homeModules`][home]
-* [`dockerImages`][docker]
 * `schemas`
+* [`templates`][templates]
+
+## Experimenting with flake schemas
+
+Flake schemas are not yet supported in Nix.
+To experiment with them, you can use the version of Nix from the [pull request][pr] to upstream.
+Here are some example commands:
```
Comment (cole-h) [inline README.md:35]: We should note that this will necessarily build Nix in its entirety before it's usable.

Comment (lucperkins) [reply README.md]: Good call. Done.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,80 @@
...
-* [`homeModules`][home]
-* [`dockerImages`][docker]
 * `schemas`
+* [`templates`][templates]
+
+## Experimenting with flake schemas
+
+Flake schemas are not yet supported in Nix.
+To experiment with them, you can use the version of Nix from the [pull request][pr] to upstream.
+Here are some example commands, though please note that the first time you run one of these commands, you will build Nix in its entirety, which is resource instensive and could take a while:
```
Comment (cole-h) [inline README.md:35]: Maybe use an admonition instead so that it stands out instead of being buried inside a standard-looking sentence.

Comment (lucperkins) [reply README.md]: Done

**Review by cole-h — APPROVED** [review]

**Pull request merged**